### PR TITLE
Completed register user and login UI tests

### PR DESCRIPTION
**Pull request by qingyuan-wu**

### CODE REVIEW

File: test/frontend/forum/test_forum.py [hunk 1]
```diff
@@ -2,43 +2,18 @@
...
-# setup, teardown in unittest https://docs.python.org/3/library/unittest.html
-def get_total_posts(driver):
-    # ensure you are navigated to the forum page when calling this function.
-    post_stats = driver.find_element(By.ID, "post-statistics-container").text
-    split_post_stats = post_stats.split()
-    return int(split_post_stats[-3]) # total posts
-
-def test_delete_post(driver):
+def test_delete_post():
+    driver = login()
```
Comment (harsimrat99) [inline test/frontend/forum/test_forum.py]: driver = login()

The username and password must be passed to this function.

File: test/frontend/login/test_login.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+import sys
+sys.path.append('../testlib')
+from register import *
+from teardown import *
+
+def test_login():
+    email = "test.account@gmail.com"
+    password = "TestAccount1@"
+    driver = login(email, password)
```
Comment (harsimrat99) [inline test/frontend/login/test_login.py]: Error: login not found.

Must be resolved by importing the **login** function from the login.py module. 

Please add "from login import *" before running test_login.

**Review by harsimrat99 — CHANGES_REQUESTED** [review]
Hi @qingyuan-wu,

Thank you for creating this pull request and for adding the tests for login and register pages. It would helpful if you could please review the feedback shared in this review.

Thank you,
Harsimrat

**Comment by qingyuan-wu** [comment]
All changes requested completed including implementing the unittest module. Conflicts with staging branch also resolved. Can merge

**Review by harsimrat99 — APPROVED** [review]

**Review by harsimrat99 — APPROVED** [review]

**Comment by harsimrat99** [comment]
Hi @qingyuan-wu ,

Thank you for making the requested changes and for your help with the testcases.

For completeness' sake I have also added the unittest classes to the login and register testcases. Thank you for your help with Selenium and frontend testcase development.

Best,
Harsimrat

**Comment by qingyuan-wu** [comment]
Thanks for implementing the unittest classes for login and register tests!

Qingyuan

**Pull request merged**